README.md

```diff
@@ -20,7 +20,7 @@ information on how to create a pull request.
 
 __Note (August 2022):__
```

Comment:
I am unsure "why" the note still says August 2022 ?

Reply:
```suggestion
__Note (July 2023):__
```